src/region.js

```diff
@@ -9,14 +9,15 @@ import isNodeAttached from './common/is-node-attached';
 import { triggerMethodOn } from './common/trigger-method';
 import MarionetteObject from './object';
 import MarionetteError from './error';
+import domMixin from './mixins/dom';
 
 const ClassOptions = [
   'allowMissingEl',
   'parentEl',
   'replaceElement'
 ];
 
-const Region = MarionetteObject.extend({
+const Region = MarionetteObject.extend(_.extend({
```

Comment:
At the moment all of the other mixins are added in an extend afterwards.  Can we do the same here and if there's a reason to change it, do it for all of them?